bigtable/spark/src/test/scala/example/WordcountLuncher.scala

```diff
@@ -0,0 +1,7 @@
+package example
```

Comment:
Why is the launcher necessary? it doesnt seem to be used

Reply:
It's to act as a test without any testing framework. It should migrate to real tests in next iteration.